.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I beleive all of these .idea files are from your IDE -- lets add .idea to the gitignore so that it doesn't show up here